lib/ckb/types/transaction.rb

```diff
@@ -62,7 +62,7 @@ def sign(key, tx_hash)
         )
       end
 
-      def to_h
+      def to_h(with_hash = true)
```

Comment:
A boolean param is not good here - what if we want to exclude another property? Then we'd have `to_h(with_hash = true, with_xxx = true)`.

I suggest we keep all properties in the model, but strip (exclude) the txhash when calling rpc.

Reply:
Thanks, it make sense, I will fix.